.github/workflows/publish.yaml

```diff
@@ -55,8 +57,8 @@ jobs:
             packages/test-apps/react-vite/dist
             packages/test-apps/nextjs/.next
           key: build-cache-${{ github.run_id }}-${{ github.run_attempt }}
-      - name: Authenticate with private GH package
-        run: echo -e "//npm.pkg.github.com/:_authToken=${{ steps.vault.outputs.GITHUB_PACKAGES_WRITE_TOKEN }}\n@contentful:registry=https://npm.pkg.github.com" > ./.npmrc
+      - name: Authenticate with private NPM and GH package
+        run: echo -e "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}\n//npm.pkg.github.com/:_authToken=${{ steps.vault.outputs.GITHUB_PACKAGES_WRITE_TOKEN }}\n@contentful:registry=https://npm.pkg.github.com" > ./.npmrc
```

Comment:
I'm wondering a bit why this worked before without explicitly adding the token. Were you trying out something or is there a deeper reasoning to that?

Reply:
This might be related to rebase, I haven't touched this file. I am gonna sync this and the feature branch with development and hopefully this diff will go away